src/v2/view-builder/views/custom-app/ChallengeCustomAppResendPushView.js

```diff
@@ -13,6 +17,25 @@ const Body = BaseForm.extend(Object.assign(
     save() {
       return loc('oie.custom_app.push.resend', 'login');
     },
+
+    showMessages() {
+      let options = {};
+       if (this.options.appState.containsMessageWithI18nKey(CUSTOM_APP_UV_ENABLE_BIOMETRIC_SERVER_KEY)) {
+        // add a title for Custom App enable biometrics message during verification
+        options.content = null;
+        options.className = 'okta-verify-uv-callout-content';
+        options.title = loc('oie.authenticator.custom_app.method.push.verify.enable.biometrics.title', 'login', [this.options.appState.getAuthenticatorDisplayName()]),
```

Comment:
i think linting will fail in this file. pls fix it.
run `yarn lint` in your local to identify the issues.